content/dstu2/operation-definition.md

```diff
@@ -0,0 +1,23 @@
+---
+title: OperationDefinition | DSTU 2 API
+---
+
+# OperationDefinition
+
+* TOC
+{:toc}
+
+## Retrieve by id
+
+List an individual OperationDefinition by its id:
+
+    GET /OperationDefinition/:id
+
+_List of OperationDefinitions_
+
+* `binary-autogen-ccd-if`: Generate Continuity of Care Document (CCD)
```

Comment:
Do not think we should call out docref as that is not really hosted by our server which this is documenting.